langchain/vectorstores/chroma.py

```diff
@@ -148,7 +148,7 @@ def similarity_search_by_vector(
             List of Documents most similar to the query vector.
         """
         results = self._collection.query(
-            query_embeddings=embedding, n_results=k, where=filter
```

Comment:
Good catch on the wrong arg name. I'm pretty sure this one is right though since it is delegating to the underlying `Collection.query` which expects an arg of the shape being passed here to appear in the `where` arg. This was the cause of my originally passing the wrong arg name above as well.

Reply:
yup, this is querying the udnelrying chroma collection, which uses `where` as a kwarg https://docs.trychroma.com/usage-guide

Reply:
Ya silly of me, I think I went too much into Find Replace on this.